### PR TITLE
chore: Update copyright notices

### PR DESCRIPTION
**Pull request by jimblanc**
#### Description of changes

Just in time for 2023, updates all of our copyright notices.

#### Checklist

- [x] PR description included
- [x] `yarn test` passes
- [ ] Tests are [changed or added](https://github.com/aws-amplify/amplify-js/blob/main/CONTRIBUTING.md#steps-towards-contributions)
- [ ] Relevant documentation is changed or added (and PR referenced)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by cshfang — APPROVED** [review]

**Review by israx — APPROVED** [review]

File: packages/analytics/src/vendor/dom-utils/closest.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2017, Philip Walton <philip@philipwalton.com>
+ * Copyright (c) 2017-2022, Philip Walton <philip@philipwalton.com>
```
Comment (elorzafe) [inline packages/analytics/src/vendor/dom-utils/closest.ts:2]: I think this vendor thing we shouldnt change it maybe?

File: packages/analytics/src/vendor/dom-utils/delegate.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2017, Philip Walton <philip@philipwalton.com>
+ * Copyright (c) 2017-2022, Philip Walton <philip@philipwalton.com>
  */
```
Comment (elorzafe) [inline packages/analytics/src/vendor/dom-utils/delegate.ts:3]: Maybe this vendor stuff I will not change it? what do you think?

File: packages/analytics/src/vendor/dom-utils/get-attributes.ts [hunk 3]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2017, Philip Walton <philip@philipwalton.com>
+ * Copyright (c) 2017-2022, Philip Walton <philip@philipwalton.com>
```
Comment (elorzafe) [inline packages/analytics/src/vendor/dom-utils/get-attributes.ts:2]: same not change it

File: packages/analytics/src/vendor/dom-utils/matches.ts [hunk 4]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2017, Philip Walton <philip@philipwalton.com>
+ * Copyright (c) 2017-2022, Philip Walton <philip@philipwalton.com>
```
Comment (elorzafe) [inline packages/analytics/src/vendor/dom-utils/matches.ts:2]: same not change it

File: packages/analytics/src/vendor/dom-utils/parents.ts [hunk 5]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2017, Philip Walton <philip@philipwalton.com>
+ * Copyright (c) 2017-2022, Philip Walton <philip@philipwalton.com>
```
Comment (elorzafe) [inline packages/analytics/src/vendor/dom-utils/parents.ts:2]: same not change it

File: packages/analytics/src/vendor/dom-utils/parse-url.ts [hunk 6]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2017, Philip Walton <philip@philipwalton.com>
+ * Copyright (c) 2017-2022, Philip Walton <philip@philipwalton.com>
```
Comment (elorzafe) [inline packages/analytics/src/vendor/dom-utils/parse-url.ts:2]: same not change it

File: packages/aws-amplify/src/index.ts [hunk 7]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```
Comment (elorzafe) [inline packages/aws-amplify/src/index.ts:2]: why not 2019-2022?

**Review by elorzafe — CHANGES_REQUESTED** [review]
I think the vendor files we shouldnt change it? but I am open to your suggestions. One minor issue on one file from `aws-amplify`

**Review by elorzafe — APPROVED** [review]
Looks good!

Thanks @jimblanc 🔥

**Pull request merged**